scipy/_lib/_array_api.py

```diff
@@ -605,6 +605,14 @@ def xp_real(x: Array, /, *, xp: ModuleType | None = None) -> Array:
     xp = array_namespace(x) if xp is None else xp
     return xp.real(x) if xp.isdtype(x.dtype, 'complex floating') else x
 
+def xp_conj(x: Array, /, xp: ModuleType | None = None) -> Array:
```

Comment:
This is fixed upstream now (see https://github.com/data-apis/array-api-compat/issues/173), so this can be dropped. Let's update `array-api-compat` instead (separate PR maybe?).

Reply:
could you address this @ev-br ?

Reply:
This is resolved, array-api-compat is updated and xp_conj is gone